download_and_create_reference_datasets/v03/write_combined_reference_ht.py

```diff
@@ -55,6 +48,9 @@ def run(environment: str, genome_version: str, dataset: str):
     else:
         ht = join_hts(DATASETS, VERSION, reference_genome=genome_version)
     ht.describe()
+    checkpoint_path = f"{GCS_PREFIXES[('dev', AccessControl.PUBLIC)]}/{uuid.uuid4()}.ht"
```

Comment:
Purely style choice but I typically like path defined up top so if we ever update them, we know where to look but this is totally opinion.

Reply:
Yeah that makes sense.  

I think this is pretty jank as is (`write_ht` should probably just do the checkpointing internally), but can fix downstream.